### ISSUE TITLE
Export to GLB file changes the scale of bones and mesh

### ISSUE THREAD

**Issue by dhaval85** [KhronosGroup/glTF-Blender-IO#1062]
**Describe the bug**
After importing FBX file with bones and exporting it as GLB file makes bones shrink. See the attached FBX file as an example. 

**To Reproduce**
1. Create a new Blender project. 
2. Import the attached FBX file by going to File->Import->FBX
3. Now export the blender project as GLB file by going to File->Export->GLB
4. Create a new Blender project. 
5. Import the exported GLB file and see the bones and everything is shrink. 

**Expected behavior**
GLB file should keep the size of bones and mesh as it is. 

**Screenshots**
<img width="803" alt="Screenshot 2020-05-07 at 10 15 17 PM" src="https://user-images.githubusercontent.com/16663529/81363077-716f8800-90b0-11ea-96b1-6d37c34ed697.png">

**.blend file/ .gltf**

[Canonical_Face_Mesh.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/4596661/Canonical_Face_Mesh.zip)

**Version**
 - OS: MacOS 10.15.4
 - Blender Version: 2.82a (2.82a 2020-03-12)

**Additional context**
NA

**Comment by donmccurdy** [comment]
See https://blender.stackexchange.com/a/109976/43930 for some context here. In glTF, and in many other programs, bones are considered "joints" that do not have a length except the distance to the next joint. Since the bones above are leaves (not connected to anything) there is no persistent concept of their length outside of Blender.

**Comment by donmccurdy** [comment]
If your use case requires a stable bone length for something, consider adding an additional bone to the end of each.

**Comment by dhaval85** [comment]
@donmccurdy  Thank you for the reply. I see your point. I see the bone distance and everything seems correct. It's the mesh which is getting bigger. I am setting up Length scale to centimetre in my project. But I guess the mesh is bigger. Do you think I can keep the scale as it is? I am applying all the transformations before exporting to GLB. 

I cannot add more bones because this is going to be used for Android ARCore Face Augmentation asset. 

Thank you!

**Comment by donmccurdy** [comment]
>  But I guess the mesh is bigger. Do you think I can keep the scale as it is? I am applying all the transformations before exporting to GLB.

Could you share a `.blend` in the final state before you would export? My guess is that the scale change is coming from something in the FBX file, it's pretty common to get a 100x multiplier with FBX. Note that glTF's units are in meters, perhaps your scene settings need to match that?

**Comment by dhaval85** [comment]
Here: [test.blend.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/4601070/test.blend.zip)

You are right. My scene settings are centimetre. Because ARCore suggest to have units in centimetre. See [here](https://developers.google.com/ar/develop/developer-guides/creating-assets-for-augmented-faces). 

Thank you very much. I guess it also has problem with the rotation too. Not sure if they are related.

**Comment by julienduroure** [comment]
Hello,

For units, cf #365

**Comment by dhaval85** [comment]
@julienduroure  Ahh I see. Thank you! I guess I will wait for that enhancement. Till then I will try to scale it down in my code and see if that works.

**Comment by singhsaabh92** [comment]
@dhaval85 I'm also facing the same problem. You found any solution for this?

**Comment by dhaval85** [comment]
> @dhaval85 I'm also facing the same problem. You found any solution for this?
> 
> 

Nope. I also abandoned my project. So never tried to make this work. But as suggested make sure the units are same. In case it doesn't work then you will have to scale it down after you import it.

**Comment by Caellian** [comment]
Related to #181.

I would like there to be some extension Blender could use to store bone metadata (normal, length, connected). Importing from Collada I faced a similar issue. It's not just a visual quirk though - blender bones handle `G` (translation transform keybind) differently for "Bone>Relations>Connected" bones and I'd like to export that metadata from code; also, Auto IK feature requires connected bones to work. Changing bones to connected fixes the problem (in Collada the bone chain was shifted back by 1 though, not sure if that's the case with glTF).

**Comment by julienduroure** [comment]
Hello,
There is no official extension ratified by Khronos Group about storing bone metadata (and not sure there are plans to propose one)

The glTF-Blender-IO project will only implement officially ratified extension.
